python/meterpreter/ext_server_stdapi.py

```diff
@@ -2548,3 +2553,15 @@ def stdapi_ui_get_idle_time(request, response):
     idle_time = (GetTickCount() - info.dwTime) / 1000
     response += tlv_pack(TLV_TYPE_IDLE_TIME, idle_time)
     return ERROR_SUCCESS, response
+
+@register_function_if(has_termios and has_fcntl)
+def stdapi_set_term_size(request, response):
+    channel_id = packet_get_tlv(request, TLV_TYPE_CHANNEL_ID)['value']
+    tlvs = list(packet_enum_tlvs(request, TLV_TYPE_UINT))
+    rows = tlvs[0]['value']
+    columns = tlvs[1]['value']
+    if channel_id in meterpreter.interact_channels:
```

Comment:
If the `channel_id` is valid, it seems like we should be returning some kind of error.